src/jsCallback.php

```diff
@@ -122,11 +122,13 @@ public function set($callback, $args = [])
      * A proper way to finish execution of AJAX response. Generates JSON
      * which is returned to frontend.
      *
-     * @param array|jsExpressionable $ajaxec  Array of jsExpressionable
-     * @param string                 $msg     General message, typically won't be displayed
-     * @param bool                   $success Was request successful or not
+     * @param array|jsExpressionable $ajaxec Array of jsExpressionable
+     * @param string $msg General message, typically won't be displayed
+     * @param bool $success Was request successful or not
      *
-     * @return [type] [description]
+     * @return void [type] [description]
```

Comment:
`[type] [description]` is probably unintended.

If nothing is expected to be returned, do not add phpdoc at all and add return type `: void` to the method header directly.

Reply:
yes, will remove it.